moveit_ros/visualization/planning_scene_rviz_plugin/src/planning_scene_display.cpp

```diff
@@ -575,7 +580,9 @@ void PlanningSceneDisplay::onEnable()
 {
   Display::onEnable();
 
-  addBackgroundJob(boost::bind(&PlanningSceneDisplay::loadRobotModel, this), "loadRobotModel");
+  addBackgroundJob(
+      boost::bind(&PlanningSceneDisplay::loadRobotModel, this, rclcpp::Node::make_shared("planning_scene_display")),
```

Comment:
Shouldn't we just pass the node or create a subnode?

Reply:
Not sure which one is better, what do you think .?